### ISSUE TITLE
3.5.10 create folder fails

### ISSUE THREAD

**Issue by asaage** [contao/core#8315]
3.5.10 after a fresh and clean install
attempt to create a new folder under files and under templates throws error
https://github.com/contao/core/blob/master/system/modules/core/library/Contao/File.php#L134

```log
[20-Apr-2016 20:31:34 GMT] PHP Fatal error: Uncaught exception 'Exception' with message 'Directory "templates/__new__" is not a file' thrown in /var/www/vhosts/testserver.de/system/modules/core/library/Contao/File.php on line 134
#0 /var/www/vhosts/testserver.de/system/modules/core/drivers/DC_Folder.php(1165): Contao\File->__construct('templates/__new...', true)
#1 /var/www/vhosts/testserver.de/system/modules/core/classes/Backend.php(650): Contao\DC_Folder->edit()
#2 /var/www/vhosts/testserver.de/system/modules/core/controllers/BackendMain.php(131): Contao\Backend->getBackendModule('tpl_editor')
#3 /var/www/vhosts/testserver.de/contao/main.php(20): Contao\BackendMain->run()
#4 {main}
[20-Apr-2016 20:31:59 GMT] PHP Fatal error: Uncaught exception 'Exception' with message 'Directory "files/__new__" is not a file' thrown in /var/www/vhosts/testserver.de/system/modules/core/library/Contao/File.php on line 134
#0 /var/www/vhosts/testserver.de/system/modules/core/drivers/DC_Folder.php(1165): Contao\File->__construct('files/__new__', true)
#1 /var/www/vhosts/testserver.de/system/modules/core/classes/Backend.php(650): Contao\DC_Folder->edit()
#2 /var/www/vhosts/testserver.de/system/modules/core/controllers/BackendMain.php(131): Contao\Backend->getBackendModule('files')
#3 /var/www/vhosts/testserver.de/contao/main.php(20): Contao\BackendMain->run()
#4 {main}
```

**Comment by Aybee** [comment]
I confirm.

**Comment by asaage** [comment]
renaming does not work either.
Not shure if it's a separate ticket - it fails silently.

**Comment by thomaspahl** [comment]
This is due to the fix https://github.com/contao/core/commit/a0bf86b82b98079c209a23f74bfdd9091dfcfb0e for #8295. `DC_Folder`now creates a `File` class instance to calculate `dirname`. `File` however is not intended for directories and rejects them in the constructor.

Immediate help is by using `DC_folder.php` of 3.5.9 (tested in 4 installations where I was tempted to rollback the whole to 3.5.9).

**Comment by leofeyer** [comment]
Fixed in f0ccac8f9ba26ecdc4bfb381e220494fb348dd5b.

**Comment by leofeyer** [comment]
Renaming fixed in add642e90684fdd0e970dc57325da8985086e52f.
